apps/antonclk/app.js

```diff
@@ -188,9 +188,8 @@ function draw() {
     g.drawString(dateStr, x, y);
     if (weekDay || calWeek) {
       var dowwumStr = require("locale").dow(date);
-      dowwumStr = "thursday";
       if (calWeek)
-        dowwumStr = (weekDay ? dowwumStr.substr(0,Math.min(dowwumStr.length,6)) + (dowwumStr.length>=6 ? "." : "") : "week ") + "#" + ISO8601calWeek(date);
+        dowwumStr = (weekDay ? dowwumStr.substr(0,Math.min(dowwumStr.length,6)) + (dowwumStr.length>=6 ? "." : "") : "week ") + "#" + ISO8601calWeek(date); //TODO: locale for "week"
```

Comment:
Maybe it is better that you use the default short weekday names require("locale").dow(date**, 1**) if you have no space.
```
var dowwumStr;
if  (calWeek && weekDay) {
  dowwumStr = require("locale").dow(date, 1) + " #" + ISO8601calWeek(date);
} else if (calWeek) {
  dowwumStr = "week #" + ISO8601calWeek(date);
} else {
  dowwumStr = require("locale").dow(date);
}
```

Reply:
Geat Idea!  I'll consider on next patch